.readthedocs.yml

```diff
@@ -27,6 +27,6 @@ sphinx:
 # Optional but recommended, declare the Python requirements required
 # to build your documentation
 # See https://docs.readthedocs.io/en/stable/guides/reproducible-builds.html
-  python:
-    install:
-    - requirements: doc/requirements.txt
+python:
```

Comment:
sorry, I can only comment here… but since you’re making changes anyway… At the top of the file wouldn’t you want to use the latest released Python version and perhaps also the latest LTS version of Ubuntu?

Reply:
Thanks - the current ReadTheDocs docs give a config with "ubuntu-22.04" and   Python: "3.12".  So, I'll go with Python 3.12.   It is curious that the OS is not "ubuntu-latest".  I'm OK with ubuntu 22, if that's what RTD wants...

I admit to being sort of "meh" on RTD, though I do also see advantages as part of a CI chain.

Reply:
I have never looked at it but given the flurry if commits/PRs here to get it to work it doesn't seem very user-friendly (or it's just that nobody knows how to use it 😉).

Not sure what the advantage in a CI workflow would be; just testing whether the documentation still builds we do for lmfit as well so you don't need RTD for that. And I am assuming that you're not planning to have a development version of the documentation online either.

Reply:
It's mostly that we need someone active with admin privileges to change the settings.

If GitHub pages was not used to update the docs, we'd have been forced to work out how to get rtd to build prs instead

Reply:
@andrewgsavage Thanks! 
@reneeotten It is fair to say that I did not really understand how RTD works, (including how it runs sphinx-build) and did not try to run from a private repo.  So, there was some flailing on the master branch partly to poke RTD and figure out how it works.  I suppose I could have read the docs first ;)

I get the appeal of RTD: automatically render and publish the docs at every tagged release.   

Using GitHub Pages could probably be automated. But I also do not mind the more deliberate step of building docs manually and pushing them to the gh-pages branch as part of the release process.